pom.xml

```diff
@@ -143,59 +107,25 @@
               <finalName>${ets-code}-${spec-version}</finalName>
               <outputDirectory>${project.build.directory}/site/data</outputDirectory>
               <descriptors>
-                <descriptor>src/assembly/data.xml</descriptor>
-              </descriptors>
-            </configuration>
-          </execution>
-          <execution>
-            <id>assemble_ctl</id>
-            <phase>package</phase>
-            <goals>
-              <goal>single</goal>
-            </goals>
-            <configuration>
-              <descriptors>
-                <descriptor>src/assembly/ctl.xml</descriptor>
+                <descriptor>${basedir}/src/assembly/data.xml</descriptor>
               </descriptors>
             </configuration>
           </execution>
           <execution>
-            <id>assemble_site</id>
             <phase>package</phase>
             <goals>
               <goal>single</goal>
             </goals>
             <configuration>
               <descriptors>
-                <descriptor>src/assembly/site.xml</descriptor>
+                <descriptor>${basedir}/src/assembly/site.xml</descriptor>
+                <descriptor>${basedir}/src/assembly/ctl-scripts.xml</descriptor>
               </descriptors>
             </configuration>
           </execution>
         </executions>
       </plugin>
       <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.18</version>
-        <configuration>
-          <includes>
-            <include>**/Verify*.java</include>
-            <include>**/*Tests.java</include>
-            <include>**/*Test.java</include>
-          </includes>
-        </configuration>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.2</version>
-        <configuration>
-          <source>1.7</source>
-          <target>1.7</target>
-        </configuration>
-      </plugin>
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-remote-resources-plugin</artifactId>
         <version>1.5</version>
```

Comment:
Is this plugin really required? If yes, I propose to move the version into the pluginManagement section of ets-common.